BenchmarkRoundStateDeepCopy panics

**Tendermint version** (use `tendermint version` or `git rev-parse --verify HEAD` if installed from source):

6b1fef7f4d54e58be4402a9581caae545d5ffcf1

**ABCI app** (name for built-in, URL for self-written if it's publicly available):

**Environment**:
- **OS** (e.g. from /etc/os-release): Mac OSX
- **Install tools**:
- **Others**:


**What happened**:

```
$ go test -run=None -bench=. -benchmem
panic: Invalid vote %!v(PANIC=String method: Unknown vote type) - expected BlockID to be either empty or complete

goroutine 37 [running]:
github.com/tendermint/tendermint/types.(*Vote).CommitSig(0xc00013c000, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/Users/cuonglm/go/src/github.com/tendermint/tendermint/types/vote.go:72 +0x2db
github.com/tendermint/tendermint/consensus/types.BenchmarkRoundStateDeepCopy(0xc0005d41c0)
	/Users/cuonglm/go/src/github.com/tendermint/tendermint/consensus/types/round_state_test.go:33 +0x33c
testing.(*B).runN(0xc0005d41c0, 0x1)
	/Users/cuonglm/sources/go/src/testing/benchmark.go:190 +0xbd
testing.(*B).run1.func1(0xc0005d41c0)
	/Users/cuonglm/sources/go/src/testing/benchmark.go:230 +0x57
created by testing.(*B).run1
	/Users/cuonglm/sources/go/src/testing/benchmark.go:223 +0x7d
exit status 2
FAIL	github.com/tendermint/tendermint/consensus/types	0.354s
```

**What you expected to happen**:

Benchmark runs successfully.

**Have you tried the latest version**: yes

**How to reproduce it** (as minimally and precisely as possible):

```
$ cd consensus/types
$ go test -run=None -bench=. -benchmem
```

**Logs (paste a small part showing an error (< 10 lines) or link a pastebin, gist, etc. containing more of the log file)**:

**Config (you can paste only the changes you've made)**:

**node command runtime flags**:

**`/dump_consensus_state` output for consensus bugs**

**Anything else we need to know**:

--- a/CHANGELOG_PENDING.md
+++ b/CHANGELOG_PENDING.md
@@ -132,3 +132,4 @@ program](https://hackerone.com/tendermint).
 - [types] \#4164 Prevent temporary power overflows on validator updates (joint
   efforts of @gchaincl and @ancazamfir)
 - [p2p] \#4140 `SecretConnection`: use the transcript solely for authentication (i.e. MAC)
+- [consensus/types] \#4243 fix BenchmarkRoundStateDeepCopy panics (@cuonglm)

--- a/consensus/types/round_state_test.go
+++ b/consensus/types/round_state_test.go
@@ -5,6 +5,7 @@ import (
 
 	amino "github.com/tendermint/go-amino"
 	"github.com/tendermint/tendermint/crypto/ed25519"
+	"github.com/tendermint/tendermint/crypto/tmhash"
 	"github.com/tendermint/tendermint/libs/rand"
 	"github.com/tendermint/tendermint/types"
 	tmtime "github.com/tendermint/tendermint/types/time"
@@ -18,9 +19,10 @@ func BenchmarkRoundStateDeepCopy(b *testing.B) {
 	vset, _ := types.RandValidatorSet(nval, 1)
 	commitSigs := make([]types.CommitSig, nval)
 	blockID := types.BlockID{
-		Hash: rand.RandBytes(20),
+		Hash: rand.RandBytes(tmhash.Size),
 		PartsHeader: types.PartSetHeader{
-			Hash: rand.RandBytes(20),
+			Hash: rand.RandBytes(tmhash.Size),
+			Total: 1000,
 		},
 	}
 	sig := make([]byte, ed25519.SignatureSize)

